rskj-core/src/main/java/co/rsk/peg/P2shErpFederation.java

```diff
@@ -49,4 +57,27 @@ public final Script getStandardRedeemScript() {
         }
         return standardRedeemScript;
     }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
Any difference between this and the one in ErpFederation and the one in LegacyErpFederation?

Reply:
yes, the parsing of the federation type. because the methods can be different (for example the extractStandardRedeemScript one)